config/locales/user_mailer/es.yml

```diff
@@ -151,6 +151,28 @@ es:
           oficina de correos de %{location} el %{date}.
       greeting: Hola,
       subject: No se pudo verificar su identidad en persona
+    in_person_outage_notification:
+      body:
+        closing_html: Le pedimos sinceras disculpas por el retraso en el servicio. Si
+          tiene alguna pregunta o no recibe su resultado antes del 1 de Junio,
+          <a href="%{contact_us_url}">póngase en contacto con el servicio de
+          atención al cliente de Login.gov</a>.
+        heading: Los resultados de la verificación presencial se pospondrán del 20 al 30
+          de Mayo
```

Comment:
I capitalized the month names thinking that was a mistake on the Gengo translators part, but I did some googling and month names aren't capitalized 😬 So could you make "mayo" and "junio" all lowercase?

Reply:
updated in 57072c98ce3b9960a162bdcd1abae3076e1e7eff and I also updated the screenshot